tests/model_registry/conftest.py

```diff
@@ -44,11 +49,15 @@
 
 @pytest.fixture(scope="class")
 def model_registry_namespace(request: FixtureRequest, admin_client: DynamicClient) -> Generator[Namespace, Any, Any]:
-    with create_ns(
-        name=request.param.get("namespace_name", MR_NAMESPACE),
-        admin_client=admin_client,
-    ) as ns:
+    ns = Namespace(name=MR_NAMESPACE, client=admin_client)
+    if ns.exists:
         yield ns
+    else:
+        with create_ns(
+            name=request.param.get("namespace_name", MR_NAMESPACE),
+            admin_client=admin_client,
+        ) as ns:
+            yield ns
```

Comment:
We should not do this.  Tests that creates namespace, must clean them up.

Reply:
If the namespace is supposed to be created automatically, and did not, we should fail. However, we should add a check that it is in active status before we proceed. Clean up should not happen here though, it should be part of update dsc call (I added it there, since update dsc, enabling model registry creates the ns, it should be explicitly cleaning it up.

Reply:
This code would be changing in a follow up PR based on our discussions offline. Resolving.